examples/window.rs

```diff
@@ -1,10 +1,12 @@
 extern crate winit;
+use winit::os::macos::WindowBuilderExt;
 
 fn main() {
     let mut events_loop = winit::EventsLoop::new();
 
     let _window = winit::WindowBuilder::new()
         .with_title("A fantastic window!")
+        .with_decorations(false)
```

Comment:
Not sure we want that in the example.

Reply:
yup, that was just from my testing, I've removed it